osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -178,7 +178,7 @@ private void initEncryption() {
     }
 
     private Cipher getCipherForEncryption()
-            throws NoSuchAlgorithmException, NoSuchPaddingException, InvalidKeyException {
+            throws NoSuchAlgorithmException, NoSuchPaddingException, InvalidKeyException, EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)